libs/hbb_common/Cargo.toml

```diff
@@ -40,10 +40,19 @@ toml = "0.7"
 uuid = { version = "1.3", features = ["v4"] }
 # crash, versions >= 0.29.1 are affected by #GuillaumeGomez/sysinfo/1052
 sysinfo = { git = "https://github.com/rustdesk-org/sysinfo" }
-
+thiserror = "1.0"
+httparse = "1.5"
+base64 = "0.22"
```

Comment:
`rustdesk/Cargo.toml` has base64 also, could you change to use `hbb_common::base64` in `rustdesk/src`?